.github/CODEOWNERS

```diff
@@ -394,8 +394,118 @@
 #CC# /x-pack/plugins/security_solution/ @elastic/security-solution
 
 # Security Solution sub teams
-/x-pack/plugins/timelines @elastic/security-threat-hunting
-/x-pack/plugins/lists @elastic/security-detections-response
+
+## Security Solution sub teams - Threat Hunting Investigations
+/x-pack/plugins/timelines @elastic/security-threat-hunting-investigations
+
+/x-pack/plugins/security_solution/common/search_strategy/timeline @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/common/types/timeline @elastic/security-threat-hunting-investigations
+
+/x-pack/plugins/security_solution/cypress/integration/timeline_templates @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/cypress/integration/timeline @elastic/security-threat-hunting-investigations
+/x-pack/plugins/security_solution/cypress/integration/detection_alerts @elastic/security-threat-hunting-investigations
```

Comment:
In this folder there are some tests owned by the detections alerts team.

Reply:
Who should ownership be for this then? Maybe it should be both our teams if that's possible?